lua/lazy/manage/task/plugin.lua

```diff
@@ -87,7 +87,7 @@ M.build = {
 
 M.docs = {
   skip = function(plugin)
-    return not plugin._.is_local and not plugin._.dirty
+    return not plugin._.is_local and not plugin._.dirty or not vim.uv.fs_access(plugin.dir, "W")
```

Comment:
If the plugin is not local, it is likely not read-only, in which case it may save IO costs by not checking permissions.

```suggestion
    return not plugin._.is_local and not plugin._.dirty or plugin._.is_local and not vim.uv.fs_access(plugin.dir, "W")
```